Makefile

```diff
@@ -285,13 +288,10 @@ empty_user_rf_pre_init.o: $(PATCHES_DIR)/empty_user_rf_pre_init.c
 	cp FRM_ERR_PATCH/*.a $(VENDOR_SDK_DIR)/lib/
 	@touch $@
 
-.sdk_patch_0.9.4:
```

Comment:
This is the second time, there is a rule for the patch, he is still in the Makefile.
Maybe there should be a patch for sdk_0.9.1? Then it hase to be
.sdk_patch_0.9.4: